admin/controllers/group.js

```diff
@@ -110,18 +121,19 @@ const addGroup = async (req, res, next) => {
       action: 'Add',
       group: req.body,
       validation: validationError.errors,
-      pupilsList: pupilsList
+      pupilsList,
+      selectedPupils
     })
   }
 
   try {
-    group = await groupDataService.create(group)
+    group = await groupService.create({'name': group.name}, group.pupils)
   } catch (error) {
     return next(error)
   }
 
   req.flash('info', 'New group created')
-  req.flash('groupId', group._id)
+  req.flash('groupId', group.id)
```

Comment:
This looks like an unconventional use of req.flash... what are you trying to achieve here?

Reply:
It's used to know which table element needs to be highlighted without having to pass a parameter in the URL.